content-api/content-actors/src/main/scala/org/sunbird/content/util/AcceptFlagManager.scala

```diff
@@ -0,0 +1,98 @@
+package org.sunbird.content.util
+
+import java.util
+
+import org.apache.commons.lang.StringUtils
+import org.sunbird.cache.impl.RedisCache
+import org.sunbird.common.dto.{Request, Response, ResponseHandler}
+import org.sunbird.common.exception.ResponseCode
+import org.sunbird.graph.OntologyEngineContext
+import org.sunbird.graph.nodes.DataNode
+import org.sunbird.graph.dac.model.Node
+import org.sunbird.graph.external.ExternalPropsManager
+import org.sunbird.graph.utils.ScalaJsonUtils
+import org.sunbird.managers.HierarchyManager
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object AcceptFlagManager {
+
+  def acceptFlag(request: Request)(implicit ec: ExecutionContext, oec: OntologyEngineContext): Future[Response] = {
+    DataNode.read(request).map(node => {
+      if (!StringUtils.equals(ContentConstants.FLAGGED, node.getMetadata.getOrDefault(ContentConstants.STATUS, "").asInstanceOf[String])) {
+        Future(ResponseHandler.ERROR(ResponseCode.CLIENT_ERROR, ContentConstants.ERR_INVALID_CONTENT, "Invalid Flagged Content! Content Can Not Be Accepted."))
+      } else {
+        request.getContext.put(ContentConstants.IDENTIFIER, node.getIdentifier)
+        createOrUpdateImageNode(request, node).map(imgNode => {
+          updateOriginalNode(request).map(response => {
+            if (!ResponseHandler.checkError(response)) {
+              response.getResult.put(ContentConstants.NODE_ID, node.getIdentifier)
```

Comment:
Instead of this:
response.getResult.put(ContentConstants.NODE_ID, node.getIdentifier)
Use
response.put(ContentConstants.NODE_ID, node.getIdentifier)

and also add,
response.put(ContentConstants.IDENTIFIER, node.getIdentifier)